packages/visx-wordcloud/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@visx/wordcloud",
+  "version": "0.1.0",
+  "description": "visx wordcloud",
+  "sideEffects": false,
+  "main": "lib/index.js",
+  "module": "esm/index.js",
+  "types": "lib/index.d.ts",
+  "files": [
+    "lib",
+    "esm"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/airbnb/visx.git"
+  },
+  "keywords": [
+    "visx",
+    "react",
+    "d3",
+    "visualizations",
+    "charts",
+    "svg"
+  ],
+  "author": "Cezar Craciun @_cezarcraciun",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/airbnb/visx/issues"
+  },
+  "homepage": "https://github.com/airbnb/visx#readme",
+  "peerDependencies": {
+    "react": "^16.8.0-0",
+    "react-dom": "^16.8.0-0"
```

Comment:
I think this isn't necessary because there aren't any calls to actually render to the DOM (just in the demo so only the demo needs it)